Providers/Modules/Plugins/SecurityBaseline/conf/security_baseline.conf

```diff
@@ -3,14 +3,14 @@
 <source> 
     type exec
     tag oms.security_baseline
-    command sleep 60 && /opt/microsoft/omsagent/plugin/omsbaseline -d  /etc/opt/microsoft/omsagent/conf/omsagent.d/ 
+    command sleep 60 && /opt/microsoft/omsagent/plugin/omsbaseline -d  /opt/microsoft/omsagent/plugin
     format json
 </source>
 
 <source> 
     type exec
     tag oms.security_baseline
-    command sleep 60 && /opt/microsoft/omsagent/plugin/omsbaseline -d /etc/opt/microsoft/omsagent/conf/omsagent.d/ 
+    command sleep 60 && /opt/microsoft/omsagent/plugin/omsbaseline -d /opt/microsoft/omsagent/plugin
```

Comment:
For clarity and consistency, I would recommend specifying the closing slash in lines 6 and 13 for the plugin directory path: `/opt/microsoft/omsagent/plugin/`